spec/acceptance/sequence_resetting_spec.rb

```diff
@@ -22,4 +40,75 @@
     expect(email).to eq "somebody1@example.com"
     expect(name).to eq "Joe"
   end
+
+  it "resets sequences back to their starting values for factory sequences" do
+    user1 = FactoryBot.create(:user)
+    user2 = FactoryBot.create(:user)
+
+    expect(user1.id).to eq 1
```

Comment:
I think we can get rid of some of these `expect`s. They are duplicating some of the other sequence tests.

I am imagining something more similar to the existing test:

```rb
it "resets inline sequences back to their starting value" do
  class User
    attr_accessor :email
  end

  FactoryBot.define do
    factory :user do
      sequence(:email) { |n| "somebody#{n}@example.com" }
    end
  end

  2.times do
    build(:user)
  end

  FactoryBot.rewind_sequences

  user = build(:user)
 
  expect(user.email).to eq "somebody1@example.com"
end
```